frontend/src/views/WheresMyGene/components/GeneSearchBar/components/ShareButton/utils.tsx

```diff
@@ -67,7 +69,13 @@ export const loadStateFromQueryParams = (
   // Check for version
   let version = params.get("ver");
   if (!version) version = "1";
-  const delimiter = version === "1" ? "-" : ",";
+  const delimiter = version > "1" ? "," : "_";
```

Comment:
```
let version = params.get("ver");
if (!version) version = "1";
```

can be compressed to `const version = params.get("ver") || "1"`

Reply:
Also just double checking, does version 1 use `"-"` or "_"` as delimiter? Can't tell from the code 😆 Thanks!

Reply:
oh my, thank you for catching this